preview-docs/server-studio-tools.md

```diff
@@ -0,0 +1,24 @@
+# The future of Apollo Server developer tooling
+
+This document outlines details on some coming changes to Apollo Server related to the embedded GraphQL Playground.
+
+## tldr: the short version
+In a coming version of Apollo Server we will change the default behavior of Apollo Server in non-production mode
+to stop serving the embedded version of GraphQL Playground by default and provide easier ways to connect your
+server to the free developer tooling in Apollo Studio.
```

Comment:
Maybe we could feather a bit of context in to the `tldr;`?

```suggestion
Apollo Server has been running GraphQL Playground in non-production environments by default for years to make it easy and convenient for you to query your graph. Apollo does not maintain Playground though –– it is a separate project and has been unmaintained for a while now. Last year, we built a GraphQL Query IDE into Apollo Studio that is significantly more powerful than Playground and fully maintained by the Apollo team.

In a coming version of Apollo Server we will change the default behavior of Apollo Server in non-production mode
to stop serving the embedded version of GraphQL Playground by default and provide easier ways to connect your
server to the free developer tooling in Apollo Studio instead. It will still be easy to configure Apollo Server to run with Playground, GraphiQL, or any other choice of a GraphQL Query IDE.
```